CHANGES.md

```diff
@@ -133,6 +133,9 @@ after the corresponding tracker issues.
   add_log_options_to_parser and get_log_from_options to obtain their
   local log object.
 
+- Fix a bug in checking relaxed solvability when excluding landmarks in the
+  presence of conditional effects.
+  <https://issues.fast-downward.org/issue1041>
```

Comment:
I commented on this on Discord. I guess I should have commented here. :-)

Reply:
I tried again with a more high-level and user-oriented changelog entry. Does this go in the direction you were thinking? Otherwise, I may have still not understood what you're aiming at and it would be easiest if you come up with a proposal that I can just insert here.

Reply:
Much improved, thanks!